### ISSUE TITLE
Dropdown Button isn't accessible

### ISSUE THREAD

**Issue by t1gu1** [quasarframework/quasar#6562]
**Is your feature request related to a problem? Please describe.**
The dropdown component isn't accessible.

With tabulation, when we exit the `q-list` component, we should continue in the flow.
- It should not send you into the browser tabs. (Tab)
- And it should not send you into the end of the page (Shift + tab)

https://quasar.dev/vue-components/button-dropdown

**Describe the solution you'd like**
If it works like the select component, it should be really better.
https://quasar.dev/vue-components/select
